src/app/(pages)/podcast-episodes/[slug]/page.tsx

```diff
@@ -1,30 +1,27 @@
+"use client"
```

Comment:
Why is "use client" on this page? This is causing some errors fpr me because async/await is only supported in server components, not client

Reply:
it was there when i was constructing the components, fixing it now

Reply:
fixed ✅